tests/unit/mixins/PrintHelper.spec.js

```diff
@@ -1,14 +1,17 @@
 import PrintHelper from '@/mixins/PrintHelper'
 import { localVue, mount } from '@support/testHelper'
-import * as printJobRequests from '@/api/PrintJobRequests'
-
-vi.mock('@/api/PrintJobRequests')
 
 describe('PrintHelper', () => {
   let helper, wrapper
   const selected = []
+  const testPrinterName = 'printer_1'
+
```

Comment:
My Personal preference would be to put these in the individual tests as you are only using them once. i.e. in 'handles success' test add let printJobV2Success and the mockResolvedValue.

Reply:
Yep agreed 👍